/**: # namespace and node name
  ros__parameters:
    path_data: 'path.txt'
    lookahead_distance: 0.3
    scale: 1.5
    desired_u: 1.0